### PR TITLE
Adds clipboards & folders to the loadout menu.

### PR DESCRIPTION
**Pull request by Plum-but-gayer**
## About The Pull Request

Adds clipboards and folders to the pocket loadout menu, allowing characters to spawn with their paperwork supplies ready to go. They live in the 'pocket' category. Clipboards are a very useful tool for managing paperwork as they are the only thing that lets you rename papers and they let you edit stuff directly without dumping all your stuff on the floor. However, on some maps for some jobs clipboards are annoyingly difficult to find - there's only a few on most maps and if you want more you need to order an entire bureaucracy crate! No more will detectives have to spend 20 minutes hunting down a way to organize their case notes. 

## How This Contributes To The Skyrat Roleplay Experience
Paperwork is HRP, also adds to the general design current of "eliminating the first 45 minute gear up period so people can get RPing faster." Enables renaming of clipboards and folders so paperwork types can rename them fun, lore-friendly things. 

## Proof of Testing

Ran a server, made a character, spawned them with the folder and clipboard.

<details>
<summary>Screenshots/Videos</summary>
  
image

image

</details>

## Changelog

:cl:
add: Bureaucrats rejoice! Clipboards and folders are now available in the loadout menu, under the 'Other' tab.
/:cl:

### CODE REVIEW

**Comment by Plum-but-gayer** [comment]
i added like 2 objects with 8 lines and its not telling me that my branch is behind the master so idk why I'm failing the linter on SOME of the maps

File: modular_skyrat/modules/loadouts/loadout_items/loadout_datum_pocket.dm [hunk 1]
```diff
@@ -287,7 +295,7 @@ GLOBAL_LIST_INIT(loadout_pocket_items, generate_loadout_items(/datum/loadout_ite
 
 
 /*
-*	DONATOR
+*	 DONATOR
```
Comment (nikothedude) [inline modular_skyrat/modules/loadouts/loadout_items/loadout_datum_pocket.dm]: why the space

Comment (Plum-but-gayer) [reply modular_skyrat/modules/loadouts/loadout_items/loadout_datum_pocket.dm]: the linter was being weird and i was trying to make it run again. i thought i reverted this tho

**Review by vinylspiders — APPROVED** [review]
I don't see any problems with this, looks good!

**Pull request merged**